app/controllers/v1/sessions_controller.rb

```diff
@@ -81,6 +65,29 @@ def metadata
 
     private
 
+    # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Yes I know 😬 
But, this method is fundamentally not hard to understand, and is there to avoid the "unsafe send based on user input" that was in place before and that the security scan is now complaining about.

Reply:
line 25 made the `send` safe, but I still prefer this.